firmware/net/components/wifi/Wifi.cc

```diff
@@ -124,13 +124,7 @@ esp_err_t Wifi::ScanNetworks(Vector<String>* ssids)
     res = esp_wifi_scan_get_ap_records(&num_aps, wifi_records);
     for (uint16_t i = 0; i < num_aps; ++i)
     {
-        String str;
-        char* ch = (char*)wifi_records[i].ssid;
-        while (*ch != '\0')
-        {
-            str.push_back(*ch);
-            ch++;
-        }
+        std::string str = (char*)wifi_records[i].ssid;
```

Comment:
At fist I thought this wouldn't work, but I realize now that the constructor would loop through the C string.